tests/e2e/cluster_downgrade_test.go

```diff
@@ -73,9 +80,9 @@ func testDowngradeUpgrade(t *testing.T, clusterSize int) {
 	t.Log("Downgrade enabled, validating if cluster is ready for downgrade")
 	for i := 0; i < len(epc.Procs); i++ {
 		validateVersion(t, epc.Cfg, epc.Procs[i], version.Versions{
-			Cluster: lastVersionStr,
+			Cluster: lastVersionZeroStr,
```

Comment:
Maybe we could rename it to `lastClusterVersion`? 
We could have just a separate variable for `lastStorageVersion`.

Reply:
renamed